src/bbs/keys.ts

```diff
@@ -0,0 +1,47 @@
+import { bbsGenerateKeyPair, bbsGeneratePublicKey } from '@docknetwork/crypto-wasm';
+import { BBSPlusPublicKeyG1, BBSPlusPublicKeyG2, BBSPlusSecretKey } from '../bbs-plus';
+import { BBSSignatureParams } from './params';
+
+/**
+ * `BBS` secret key.
+ */
+export class BBSSecretKey extends BBSPlusSecretKey {
```

Comment:
Makes sense because it will be possible to use it in places where  `BBSPlusSecretKey` is expected. (Same for `class BBSPublicKey extends BBSPlusPublicKeyG2`).